### PR TITLE
Update to GrimoireLab 0.9.0 and OpenSearch 2.6

### PR DESCRIPTION
**Pull request by sduenas**
This PRs updates the toolkit to use the latest versions of GrimoireLab and OpenSearch images.

### CODE REVIEW

File: ansible/roles/opensearch/tasks/security_config.yml [hunk 1]
```diff
@@ -37,9 +37,9 @@
     group: '1000'
     mode: 0644
   loop:
-    - path: securityconfig/config.yml
+    - path: opensearch-security/config.yml
       always_update: this
-    - path: securityconfig/internal_users.yml
-    - path: securityconfig/roles_mapping.yml
+    - path: opensearch-security/internal_users.yml
+    - path: opensearch-security/roles_mapping.yml
```
[the remark this answers is not in the document]
Comment (sduenas) [reply ansible/roles/opensearch/tasks/security_config.yml:43]: Done.

**Comment by zhquan** [comment]
We need to change the `server.host: "0"` to `server.host: "0.0.0.0"` in  [opensearch_dashboards.yml.j2](https://github.com/bitergia-analytics/bap-deployment-toolkit/blob/main/ansible/roles/opensearch_dashboards/templates/opensearch_dashboards.yml.j2#L2)

**Comment by sduenas** [comment]
> server.host: "0"

Done, too!

**Review by zhquan — APPROVED** [review]
LGTM

**Pull request merged**